### PR TITLE
add String2Image && fix Base64ToImage

### PR DESCRIPTION
**Pull request by MRXLT**

### CODE REVIEW

File: python/paddle_serving_app/reader/image_reader.py [hunk 1]
```diff
@@ -517,6 +517,19 @@ def __repr__(self):
...
+    def __init__(self):
+        pass
+
+    def __call__(self, img_buffer):
+        data = np.fromstring(img_buffer, np.uint8)
+        img = cv2.imdecode(data, cv2.IMREAD_COLOR)
+        return img
+
+    def __repr__(self):
+        return self.__class__.__name__ + "()"
```
Comment (guru4elephant) [inline python/paddle_serving_app/reader/image_reader.py:530]: could add a unit test for this?

File: python/paddle_serving_app/reader/image_reader.py [hunk 2]
```diff
@@ -561,7 +574,9 @@ def __init__(self):
         pass
 
     def __call__(self, img_base64):
-        img = base64.b64decode(img_base64)
+        sample = base64.b64decode(img_base64)
+        data = np.fromstring(sample, np.uint8)
+        img = cv2.imdecode(data, cv2.IMREAD_COLOR)
```
Comment (guru4elephant) [inline python/paddle_serving_app/reader/image_reader.py:579]: add a unit test for this please.

File: python/paddle_serving_app/reader/image_reader.py (hunk 1 above)
Comment (MRXLT) [reply python/paddle_serving_app/reader/image_reader.py:530]: done

File: python/paddle_serving_app/reader/image_reader.py (hunk 2 above)
Comment (MRXLT) [reply python/paddle_serving_app/reader/image_reader.py:579]: done

File: python/examples/imagenet/test_image_reader.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+from paddle_serving_app.reader.image_reader import String2Image, Base64ToImage, Sequential
+import base64
+
+
+def test_String2Image():
+    with open("./daisy.jpg") as f:
+        img_str = f.read()
+    seq = Sequential([String2Image()])
+    img = seq(img_str)
+    print(img.shape)
```
Comment (guru4elephant) [inline python/examples/imagenet/test_image_reader.py]: can you just assert the shape?

Comment (MRXLT) [reply python/examples/imagenet/test_image_reader.py]: done

**Review by guru4elephant — APPROVED** [review]
LGTM

**Pull request merged**